static/js/app/partials/organizations.html

```diff
@@ -1,12 +1,75 @@
-<div class="page-header">
-  <h3 class="text-center">{{activeOrg.entity.name}} organization</h3>
-</div>
+<div ng-switch="activeOrg">
 
-<div ng-switch="activeSpaces">
-    <div ng-switch-default>
-        <ng-include src="'js/app/partials/spaces.html'"></ng-include>
+    <div ng-switch-when="404">
+        <div class="page-header">
+            <h3 class="text-center">Sorry, we couldn't find that org</h3>
+        </div>
+    </div>
+
+    <div ng-switch-when="undefined">
+        <!-- show nothing while loading -->
```

Comment:
In the future, we could do a loading spinner. But for now, put something like "Loading..."